pkg/db/db_releases.go

```diff
@@ -134,6 +134,28 @@ func (h *DBHandler) DBSelectReleaseByVersion(ctx context.Context, tx *sql.Tx, ap
 	return h.processReleaseRow(ctx, err, rows, ignorePrepublishes, true)
 }
 
+func (h *DBHandler) DBSelectReleaseByReleaseNumbers(ctx context.Context, tx *sql.Tx, app string, releaseVersion types.ReleaseNumbers, ignorePrepublishes bool) (*DBReleaseWithMetaData, error) {
+	span, ctx := tracer.StartSpanFromContext(ctx, "DBSelectReleaseByVersion")
```

Comment:
Please use this function instead, so that errors are marked as errors in datadog (also use the "onErr" function on every return err)
```suggestion
	span, ctx, onErr := tracing.StartSpanFromContext(ctx, "DBSelectReleaseByVersion")
```

Reply:
this applies to all db functions at least